### DISCUSSION TITLE
Sharing a REX of the creation of a React lib from an existing vanilla lib

### DISCUSSION THREAD

**Discussion by dcz-switcher** [codegouvfr/react-dsfr#318] [Q&A] [answered]
Hello, I'm working in a team which looking for create a React library from an existing vanilla one. Something very similar to your project. Is there a way to contact you for having some insights and return of experience ?
Best regards

**Comment by garronej** [comment, marked as answer]
Hello @dcz-switcher,

Absolutely! Feel free to reach out to us via email at joseph.garrone@protonmail.com and dylan.decrulle@outlook.fr.

We’d love to hear more about your project and the audience you’re targeting with the library.

Best regards,
